frontend/src/components/staff/FullTable.jsx

```diff
@@ -26,6 +27,13 @@ export default function FullTable({ refresh, onRefresh }) {
         setInventoryData(dataWithKey);
         setViewData(dataWithKey);
         setLoading(false);
```

Comment:
```suggestion
```
since setLoading was set outside the fetch, it'd be called twice on a successful call